packages/wrangler/package.json

```diff
@@ -51,13 +51,11 @@
 	],
 	"scripts": {
 		"assert-git-version": "node -r esbuild-register scripts/assert-git-version.ts",
-		"build": "pnpm run clean && pnpm run bundle && pnpm run emit-types && pnpm run generate-json-schema",
-		"bundle": "node -r esbuild-register scripts/bundle.ts",
+		"build": "pnpm run clean && pnpm tsup && pnpm run generate-json-schema",
```

Comment:
This seems quite a significant change. Are there any implications of not using the custom build script?